sdk/dynatrace/Azure.ResourceManager.Dynatrace/src/Customized/DynatraceMonitorResource.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Autorest.CSharp.Core;
+using Azure.Core;
+using Azure.ResourceManager.Dynatrace.Models;
+
+namespace Azure.ResourceManager.Dynatrace
+{
+    public partial class DynatraceMonitorResource
+    {
+        /// <summary>
+        /// Gets the user account credentials for a Monitor
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Dynatrace.Observability/monitors/{monitorName}/getAccountCredentials</description>
+        /// </item>
+        /// <item>
+        /// <term>Operation Id</term>
+        /// <description>Monitors_GetAccountCredentials</description>
+        /// </item>
+        /// <item>
+        /// <term>Default Api Version</term>
+        /// <description>2021-09-01</description>
+        /// </item>
+        /// <item>
+        /// <term>Resource</term>
+        /// <description><see cref="DynatraceMonitorResource"/></description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        // Add this custom code due to the api compatibility for operation: Monitors_GetAccountCredentials.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual async Task<Response<DynatraceAccountCredentialsInfo>> GetAccountCredentialsAsync(CancellationToken cancellationToken = default)
```

Comment:
the reason that we have to introduce this back here is that in the new api-version, we no longer have this operation.
Therefore we introduce it back as back compat.
Is this correct?